docs/integrations/source_version_control.md

```diff
@@ -22,11 +22,27 @@ Feel free to switch out the `rev` value to something else, like another
 branches or other mutable refs since the hook [won't auto update as you may
 expect][pre-commit-mutable-rev].
 
-If you want support for Jupyter Notebooks as well, then replace `id: black` with
-`id: black-jupyter`.
+## Jupyter Notebooks
+
+If you want to include support for Jupyter Notebooks, then simply replace the hook's
+`id: black` with `id: black-jupyter` in the `.pre-commit-config.yaml`:
 
 ```{note}
-The `black-jupyter` hook is only available from version 21.8b0 and onwards.
+The `black-jupyter` hook became available in version 21.8b0.
```

Comment:
Let's put this note below the yaml sample.

Reply:
Yeah I was thinking about this too.  Moved it down 👍